src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -114,54 +114,48 @@ The "Save Gantt" button will take a screenshot of the visible part of the Gantt
 
 Job Planner uses a  Calendar Definition to know how the job will be planned over the time. As shown on the example below, this definition is composed of 4 fields:
 
- - a description (saying what the cron expression means, when to use the Calendar Definition, etc.)
+ - a description (providing humain-readable description of the used cron expression, when is the Calendar Definition used, etc.)
+ - a time zone to define a specific time zone for the future scheduled jobs depending on the region (Europe/Paris, US/Eastern, etc.)
```

Comment:
please check the repetition here "a time zone to define a specific time zone". At least, if the first used `time zone` in the sentence is a button or something refer to it using capitals.

Reply:
done